conference/models.py

```diff
@@ -1595,7 +1595,7 @@ class StripePayment(models.Model):
         ('FAILED', 'Failed'),
     )
 
-    uuid = models.CharField(max_length=40, unique=True)
+    uuid = models.CharField(max_length=40, unique=True, default=uuid.uuid4)
```

Comment:
Another possible option here would be to use short uuid, I think we already use it for Talks for example in the same module; (unless we already used uuid4 somewhere else in payments, then this default obviously makes more sense). WDYT?

Reply:
We've been using `uuid4` for this until now - https://github.com/EuroPython/epcon/blob/ep2019/conference/cart.py#L170. I'd keep it as-is